### PR TITLE
Feature/1761/select audio output

### LINKED ISSUE #1721: Disable proximity sensor for video calls
Currently WhatsApp, google due have this feature while on a video call the camera or screen dose not turn off is very frustrating when going over the proximity sensor and camera turns off and screen turns off and sometimes it dose not come back on. Very similar to  #758, #920, #638. 

Please anyway we can get an option to disable the proximity sensor!! 🙏

### LINKED ISSUE #1761: Allow to select audio output channel
It should be possible to select the audio output (speaker, earphones, bluetooth, ....)

Ref https://github.com/nextcloud/talk-ios/issues/712

### PR DESCRIPTION
**Pull request by mahibi**
resolve #1761 
resolve #1721 

please test extensively

standard call view | audio output selection | audio output selection with wired headset
------- | -------- | --------
grafik | grafik | grafik

### CODE REVIEW

**Comment by mahibi** [comment]
TODO:
make the "arrow down" icon for current audio output selection to indicate that there are options to choose. 
maybe duplicate icons?! 

.. i suggest to do this in a seperate PR and merge this first..

**Comment by AndyScherzinger** [comment]
> .. i suggest to do this in a seperate PR and merge this first..

makes sense and yes for starters it probably be easier if you duplicate the icons and add the arrow, else you might need a custom UI element.

**Review by timkrueger — APPROVED** [review]

File: app/src/main/java/com/nextcloud/talk/activities/CallActivity.java [hunk 1]
```diff
@@ -378,7 +378,7 @@ private void createCameraEnumerator() {
         try {
             camera2EnumeratorIsSupported = Camera2Enumerator.isSupported(this);
         } catch (final Throwable throwable) {
-            Log.w(TAG, "Camera2Enumator threw an error");
+            Log.w(TAG, "Camera2Enumerator threw an error");
```
Comment (AndyScherzinger) [inline app/src/main/java/com/nextcloud/talk/activities/CallActivity.java:381]: should we log the actual throwable since it is missing?

**Comment by AndyScherzinger** [comment]
@mahibi I do have 2 minor changes for the bottom sheet(s) should I push them on top of this ?

**Review by AndyScherzinger — APPROVED** [review]

**Comment by AndyScherzinger** [comment]
@mahibi DONE :) Login Test fails while that is unrelated to my changes, so maybe a server side hiccup.

**Comment by AndyScherzinger** [comment]
rebase won't be triggered since branch is up-to-date with master, so I bumped the detekt lib version. Let's see :)

**Comment by AndyScherzinger** [comment]
My pleasure 😊

**Pull request merged**